hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/MarkerFiles.java

```diff
@@ -135,6 +135,7 @@ public boolean doesMarkerDirExist() throws IOException {
     if (subDirectories.size() > 0) {
       parallelism = Math.min(subDirectories.size(), parallelism);
       SerializableConfiguration serializedConf = new SerializableConfiguration(fs.getConf());
+      context.setJobStatus(this.getClass().getSimpleName(), "MarkerFiles created and merged data paths");
```

Comment:
change to : `Obtaining marker files for all created, merged paths`

Reply:
done